WebNetSample/Controllers/ProductsController.cs

```diff
@@ -18,4 +23,47 @@ public async Task<IActionResult> Index()
 
         return View(productsWithDetails);
     }
+
+    public async Task<IActionResult> AllProducts()
+    {
+        var productsWithDetails = await _productService.GetListAsync(_paginationParameters);
+
+        return View(productsWithDetails);
+    }
+
+    [HttpGet]
+    public IActionResult Add()
+    {
+        Guid id = Guid.NewGuid();
+        DateTime date = DateTime.Now;
+
+        ViewBag.Id = id;
```

Comment:
Well, I would prefer to use strictly typed view since `ViewBag` is really unreliable guy